ci/provisioning/post_provision_config_common_functions.sh

```diff
@@ -192,9 +192,9 @@ set_local_repo() {
     version="$(lsb_release -sr)"
     version=${version%%.*}
     if [ "$repo_server" = "artifactory" ] &&
-       [ -z "$(rpm_test_version)" ] &&
-       [[ ${CHANGE_TARGET:-$BRANCH_NAME} != weekly-testing* ]] &&
-       [[ ${CHANGE_TARGET:-$BRANCH_NAME} != provider-testing* ]]; then
+       [ -z "$CI_RPM_TEST_VERSION" ] &&
```

Comment:
This is ignoring any commit pragma value.  This needs to be reverted back to use the function and the function updated to account for the parameter.

Reply:
It handles it properly,  the post_provision_config.h change above puts the commit pragma if it exists into the CI_RPM_TEST_VERSION environment variable, if the CI_RPM_TEST_VERSION parameter is not set.

Reply:
Fixed in current push.